lib/phantomjs.js

```diff
@@ -61,3 +63,34 @@ if (exports.path) {
     // We did our best. Likely because phantomjs was already installed.
   }
 }
+
+/**
+ * Executes a script or just runs PhantomJS
+ */
+exports.exec = function () {
+  var args = Array.prototype.slice.call(arguments)
+  return spawn(exports.path, args)
+}
+
+/**
+ * Runs PhantomJS with provided options
+ * @example
+ * // handy with WebDriver
+ * phantomjs.run('--webdriver=4444').then(program => {
+ *   // do something
+ *   program.kill()
+ * })
+ * @returns {Promise} the process of PhantomJS
+ */
+exports.run = function () {
+  var program = exports.exec.apply(null, arguments)
```

Comment:
Yeah, for example, `--bogus` makes it hang (without error messages). Is it better to be guarded [like this](https://github.com/cognitom/gulp-sketch/blob/master/coffee/index.coffee#L44-L55)?
http://phantomjs.org/api/command-line.html